src/lib/services/submit.services.ts

```diff
@@ -210,6 +211,72 @@ export const submitMotionProposal = async ({
 	});
 };
 
+export const submitAddNodeProviderProposal = async ({
+	neuronId,
+	governance,
+	...rest
+}: {
+	user: UserOption;
+} & Partial<Pick<ProposalParams, 'neuronId' | 'governance'>>): Promise<SubmitProposalResult> => {
+	const submit = async ({
+		metadata,
+		neuronId
+	}: { metadata: ProposalEditableMetadata } & Pick<
+		ProposalParams,
+		'neuronId'
+	>): Promise<SubmitProposalResult> => {
+		const { url, nodeProviderId } = metadata;
+
+		const title = 'Add Node Provider: ' + metadata?.nodeProviderName;
+		const summary =
+			'Register a node provider' +
+			metadata?.nodeProviderName +
+			' in line with the announcement and discussion at ' +
+			metadata?.url +
+			'. \n\nThe self-declaration documentation is available at ' +
+			metadata?.urlSelfDeclaration +
+			' \nwith SHA256 hash: ' +
+			metadata?.hashSelfDeclaration +
+			'.\n\nThe proof of identity is available at ' +
+			metadata?.urlIdentityProof +
+			' \nwith SHA256 hash: ' +
+			metadata?.hashIdentityProof +
+			'.';
+
+		if (isNullish(title) || isNullish(url) || isNullish(nodeProviderId) || isNullish(summary)) {
```

Comment:
tite and url cannot be null or undefined here given that you just set those above. not sure what that checks is meant for then?